apt/private/deb_import.bzl

```diff
@@ -0,0 +1,29 @@
+"deb_import"
+
+load("@bazel_tools//tools/build_defs/repo:http.bzl", "http_archive")
+
+# BUILD.bazel template
```

Comment:
nit: maybe it should go in `BUILD.deb` following typical convention?

Reply:
Hmm, it could go into a file and passed to http_archive build_file attribute. I'll reconsider it in the future.